src/test/java/org/javarosa/core/model/utils/test/LocalizerTest.java

```diff
@@ -680,10 +680,15 @@ public void run() {
     private void runAsync(Runnable test, String label) {
         Thread t = new Thread(test);
         t.start();
-        try {
-            t.join(50);
-        } catch (InterruptedException e) {
+        int attempts = 4;
```

Comment:
This just makes this test get a bit longer to run. Weird things were causing it to not finish in the 50ms for me on occasion (I think garbage collection on the second thread)